### ISSUE TITLE
status: fail reason: old_version

### ISSUE THREAD

**Issue by Aki92** [tgalal/yowsup#1367]
Hey Team,

I installed the latest version of Yowsup but I am getting the below mentioned issue which is marked everywhere that it's fixed in current version. I also tried installing wa16 version but getting the same issue:

Command I ran:
**./yowsup-cli registration --requestcode sms --phone 9184471***** --cc 91 --mcc 404 --mnc 11**

Error:
**INFO:yowsup.common.http.warequest:{"status":"fail","reason":"old_version"}

status: fail
reason: old_version**

Hoping to get it resolved ASAP.

**Comment by neerajvashistha** [comment]
Same problem here.
```
~/yowsup$ yowsup-cli registration --requestcode sms --phone 91xxxxxxxxxx -C 91 -m 405 -n 037
INFO:yowsup.common.http.warequest:{"status":"fail","reason":"old_version"}

INFO:yowsup.common.http.warequest:{"status":"fail","reason":"old_version"}

status: fail
reason: old_version
```
went through issue #762 , #1138 , #1184 . 
~/yowsup$ yowsup-cli demos -v
yowsup-cli v2.0.13
Using yowsup v2.4.48
tried to upgrade by uninstalling using pip then git clone.. setup.py install but same result.
 
problem still persist. Any help would be appreciated.

**Comment by neerajvashistha** [comment]
cheers, this issue can be solved by reading the issue #1335, solution is changing the cloned files as described by @najkirdneh in pull req by [jlguardi](https://github.com/tgalal/yowsup/pull/1336/files) then building the yowsup( uninstall first ) again by

`python setup.py install`

**Comment by Aki92** [comment]
Yup @neerajvashistha, my problem also got solved by the pointed solution. Thanks a lot for your help :+1:
